games/urls/achievements.py

```diff
@@ -0,0 +1,12 @@
+# pylint: disable=C0103
+from __future__ import absolute_import
+from django.conf.urls import url
+from games.views import achievements as views
+
+
+urlpatterns = [
+    url(r'^/unlock/(?P<achievement_id>\d+)$', views.UnlockAchievementView.as_view(),
+        name='api_game_list'),
```

Comment:
be sure to set unique names for each route, in this case it would be something like `api_achievement_unlock`